packages/core/types/global.d.ts

```diff
@@ -7,3 +7,34 @@ declare module 'navi-core/templates/*' {
 
 type Dict<T = string> = { [key: string]: T };
 type TODO<T = any> = T;
+
+type FragmentInstance = TODO;
+declare module 'ember-data-model-fragments/fragment' {
+  import DS from 'ember-data';
+  export default DS.Model; //TODO make me real
+}
+
+declare module 'ember-data-model-fragments/FragmentArray' {
+  import MutableArray from '@ember/array/mutable';
+  type BaseEnumerable<T> = Array<T> & MutableArray<T>;
+  export default interface FragmentArrayBase<T extends FragmentInstance> extends BaseEnumerable<T> {
+    pushObject(fragment: FragmentInstance): TODO;
+    removeFragment(fragment: FragmentInstance): TODO;
+  }
+}
+
+declare module 'ember-cp-validations' {
+  type EmberCPValidationsMixin = {
+    validations: TODO;
+  };
+  function buildValidations(validation: TODO): EmberCPValidationsMixin;
+  function validator(name: string, options?: TODO): TODO;
+  export { buildValidations, validator };
+}
+
+declare module '@ember-data/store' {
```

Comment:
We should be able to update the `ts-ignore`s here
https://github.com/yahoo/navi/blob/ec0173733793f705b86e9f1e18ad5e86c8b83ac2/packages/core/types/ember-data/store.d.ts
It would be nice if we copied the same behavior (and maybe create a public registry)

Reply:
Ah, didn't know we hadt this already. Yeah, I wanted to come back and make the fragment registery.

Reply:
https://github.com/yahoo/navi/issues/971